index.html

```diff
@@ -1,11 +1,370 @@
+                  <h3>Hôtel Le soleil du matin</h3>
+                  <p>Nuit à partir de 128&euro;</p>
+                  <ul>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                  </ul>
+                </figcaption>
+              </figure>
+            </a>
+          </article>
+
+          <article>
+            <a href="#">
+              <figure>
+                <img
+                  src="./images/hebergements/4_small/aw-creative-VGs8z60yT2c-unsplash.jpg"
+                  alt="Grande chambre avec un lit double, un haut plafond et une porte qui mène vers une autre pièce."
+                />
+                <figcaption>
+                  <h3>Au cœur de l'eau Chambre d'hôtes</h3>
+                  <p>Nuit à partir de 71&euro;</p>
+                  <ul>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star"></i></li>
+                  </ul>
+                </figcaption>
+              </figure>
+            </a>
+          </article>
+
+          <article>
+            <a href="#">
+              <figure>
+                <img
+                  src="./images/hebergements/4_small/febrian-zakaria-sjvU0THccQA-unsplash.jpg"
+                  alt="Chambre d'hôtel avec un lit double."
+                />
+                <figcaption>
+                  <h3>Hôtel Tout bleu et Blanc</h3>
+                  <p>Nuit à partir de 68&euro;</p>
+                  <ul>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star active"></i></li>
+                    <li><i class="fa-solid fa-star"></i></li>
+                  </ul>
+                </figcaption>
+              </figure>
+            </a>
+          </article>
+        </section>
+      </div>
+
+      <!-- Activités cards-->
+
+      <div id="activities">
+        <section>
+          <h2><strong>Activités à Marseille</strong></h2>
+          <figure>
+            <img
+              src="./images/activites/4_small/reno-laithienne-QUgJhdY5Fyk-unsplash.jpg"
+              alt="Le vieu Port de Marseille bondé de bateaux et de quelques voiliers"
+            />
+
+            <figcaption>
+              <h3>Vieux Port</h3>
+            </figcaption>
+          </figure>
```

Comment:
Use article same as accomodations and popular. Then put first article inside a div, the next 2 an 3 on other div, 4 in a div and 5 & 6 in a div